docs/supported-metrics.md

```diff
@@ -54,3 +54,48 @@ In Mesmer we support 3 types of metrics:
 | Stream throughput            | counter |
 | Operator throughput          | counter |
 | Operator processing time     | counter |
+
+## ZIO (both Default-JVM-Metrics and others)
+
+| Metric                                | Type    |
+|---------------------------------------|---------|
+| Executor Capacity                     | gauge   |
+| Executor Concurrency                  | gauge   |
+| Executor Dequeued Count               |         |
+| Executor Enqueued Count               |         |
+| Executor Size                         | gauge   |
+| Executor Worker Count                 |         |
+| JVM Buffer Pool Capacity Bytes        | gauge   |
+| JVM Buffer Pool Used Buffers          | gauge   |
+| JVM Buffer Pool Used Bytes            | gauge   |
+| JVM Classes Loaded                    | gauge   |
+| JVM Classes Loaded Total              | gauge   |
+| JVM Classes Unloaded Total            | gauge   |
+| JVM GC Collection Seconds Count       |         |
+| JVM GC Collection Seconds Sum         |         |
```

Comment:
Histogram?

Reply:
if that's a histogram then there's _count, _sum and _buckets metric in prometheus.

Reply:
@lgajowy, refactored in this commit:
https://github.com/ScalaConsultants/mesmer/pull/470/commits

Based off my testing (as seen in below screenshot), I could not find any `ZIO DefaultJvm _buckets metric in prometheus.` 🙁